VERSION

```diff
@@ -1 +1 @@
-0.5.3
+0.5.4
```

Comment:
Still a patch here, please bump a minor given that there's user facing features added
```suggestion
0.6.0
```